lib/rabbitmq/http/client/version.rb

```diff
@@ -1,7 +1,7 @@
 module RabbitMQ
   module HTTP
     class Client
-      VERSION = "1.7.0.pre1"
+      VERSION = "1.7.1.pre1"
```

Comment:
Never bump versions in pull requests. It's not up to you to decide when it's time to do so.